### PR TITLE
Day1

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #316

## Changes proposed

request changes

## Screenshots
Screenshot (748)
Screenshot (749)
Screenshot (750)

### CODE REVIEW

**Comment by parma1111** [comment]
Now you can review my PR

**Comment by rupali-codes** [comment]
please resolve conflicts

**Comment by parma1111** [comment]
now you can review the PR

**Comment by parma1111** [comment]
Now you can review the PR

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -55,12 +55,20 @@
       "language": "english"
     },
     {
-        "name": "CodeWithHarry",
-        "description": "This channel provides free Web Development course for beginners.",
-        "url": "https://www.youtube.com/@CodeWithHarry/playlists",
+      "name": "Thenewboston",
+      "description": "The new boston is a great channel for learning docker,blockchain,webdevelopment,Django and Angular2 etc.",
```
Comment (CBID2) [inline database/youtube/web-development.json:59]:
```suggestion
      "description": "The new boston is a great channel for learning docker,blockchain,webdevelopment,Django, Angular2, and etc.",
```

File: database/youtube/web-development.json [hunk 2]
```diff
@@ -78,5 +86,4 @@
         "subcategory": "web-development",
         "language": "english"
     }
```
Comment (CBID2) [inline database/youtube/web-development.json:88]: Don’t forget to add a description! 😊

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions @parma1111! 😊

File: database/youtube/web-development.json (hunk 2 above)
Comment (parma1111) [reply database/youtube/web-development.json:88]: codewithharry already exist in project so I have removed from my PR and other changes you told I have done that

**Comment by parma1111** [comment]
codewithharry link is already exist that's why I have removed from my PR and other changes you told that I have done.

**Review by CBID2 — APPROVED** [review]
Looks great @parma1111! 😄

**Comment by rupali-codes** [comment]
> codewithharry link is already exist that's why I have removed from my PR and other changes you told that I have done.

I can not see it, could please screenshot that?

**Comment by parma1111** [comment]
> > codewithharry link is already exist that's why I have removed from my PR and other changes you told that I have done.
> 
> I can not see it, could please screenshot that?

> Now you can see and review

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**